src/mail/editor/SendMailModel.ts

```diff
@@ -878,55 +885,78 @@ export class SendMailModel {
 		return !isSecurePassword(minimalPasswordStrength)
 	}
 
+	saveDraft(
+		saveAttachments: boolean,
+		mailMethod: MailMethod,
+		blockingWaitHandler: (arg0: TranslationKey | lazy<string>, arg1: Promise<any>) => Promise<any>,
```

Comment:
I don't remember why we have this weird handler thing? Can't we just call progressDialog or whatever on the calling side with the returned promise?

Also with this recent change it will call it multiple times now which is probably not what we want?